website/addons/onedrive/README.md

```diff
@@ -0,0 +1,8 @@
+# OSF OneDrive Addon
+
+Enabling the addon for development
+
+1. In `website/settings/local.py` add, `"onedrive"` to the `ADDONS_REQUESTED` list.
```

Comment:
Looks like you've added `onedrive` to addons.json, which should take care of this step automatically.